pkgs/tools/security/vaultwarden/default.nix

```diff
@@ -1,10 +1,10 @@
 { lib, stdenv, callPackage, rustPlatform, fetchFromGitHub, nixosTests
 , pkg-config, openssl
 , libiconv, Security, CoreServices, SystemConfiguration
-, dbBackend ? "sqlite", libmysqlclient, postgresql }:
+, dbBackend ? "sqlite", libmysqlclient, postgresql, python3 }:
 
 let
-  webvault = callPackage ./webvault.nix {};
+  webvault = callPackage ./webvault.nix { inherit python3; };
```

Comment:
We don't need to pass python through here and it kinda hides why we actually need to overwrite it.